docs/source/release/v6.12.0/reflectometry.rst

```diff
@@ -7,10 +7,14 @@ Reflectometry Changes
 
 New Features
 ------------
-.. amalgamate:: Reflectometry/New_features
+- :ref:`algm-SaveISISReflectometryORSO` now saves the spin state for group workspaces that are subject to polarisation efficiency correction algorithms.
+- New algorithm :ref:`algm-ReflectometryISISCreateTransmission` for creating transmission workspaces that are suitable for passing to :ref:`algm-PolarizationEfficienciesWildes`.
 
 Bugfixes
 --------
-.. amalgamate:: Reflectometry/Bugfixes
+- ``Polarization Correction`` and ``Flood Correction`` text boxes in the :ref:`Experiment Settings<refl_exp_instrument_settings>` tab of the :ref:`Reflectometry Interface<interface-isis-refl>` are now more robust to file path inputs (e.g. will no longer crash when ``"C:"`` is entered).
+- :ref:`algm-ReflectometryReductionOneLiveData` will no longer re-write the spectra map when loading the instrument into the input live data workspace.
+- :ref:`algm-ReflectometryReductionOneLiveData` now retrieves the correct exponential correction parameters from the instrument definition file.
```

Comment:
Sorry, I failed to spot this on the original PR - the algorithm referenced in this bug fix for the exponential correction parameters should be `ReflectometryReductionOneAuto`.